pkg/commands/componentarchive/signature/verify/verify.go

```diff
@@ -50,9 +50,6 @@ type GenericVerifyOptions struct {
 	// SignatureName selects the matching signature to verify
 	SignatureName string
 
-	// SkipAccessTypes defines the access types that will be ignored for verification
-	SkipAccessTypes []string
```

Comment:
I'm not sure if the access type is really the relevant selection criteria. Assume you have different references to documents stored in S3 and some of them are important but others not. Furthermore, what if the repository type of a resource is modified during a transport getting another access type e.g. when the target landscape does not have an S3 store?

Reply:
as discussed, the skip access types is just a selection criteria for the cli, in the component-spec it is defined as exclude from signature digest.